rubicon_ml/sklearn/pipeline.py

```diff
@@ -131,6 +131,35 @@ def score(self, X, y=None, sample_weight=None):
 
         return score
 
+    def score_samples(self, X, experiment=None):
+        """Score with the final estimator and automatically
+        log the results to Rubicon.
+
+        Parameters
+        ----------
+        X : iterable
+            Data to predict on. Must fulfill input requirements of first step of the pipeline.
+        y : iterable, optional
+            Targets used for scoring. Must fulfill label requirements for all steps of the pipeline.
+        sample_weight : list, optional
+            If not None, this argument is passed as sample_weight keyword argument to the
+            score method of the final estimator.
+        """
+        if experiment is None:
+            experiment = self.project.log_experiment(**self.experiment_kwargs)
```

Comment:
I think we should follow the same logic as `score` and check `self.experiment` too